### PR TITLE
Remove py

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: pytest_django/plugin.py [hunk 1]
```diff
@@ -120,22 +113,30 @@ def _handle_import_error(extra_message):
...
+
+        if not args:
+            args = [pathlib.Path.cwd()]
+
+        for arg in args:
+            if is_django_project(arg):
+                return arg
+            for parent in arg.parents:
+                if is_django_project(parent):
+                    return parent
```
Comment (blueyed) [inline pytest_django/plugin.py]: @voidus 
This (looking at parent) is a behavior change already, isn't it?
(at least it is not covered with tests)

Comment (blueyed) [reply pytest_django/plugin.py]: Oh, sorry.. `arg.parts(reverse=True)` handled this before

**Comment by blueyed** [comment]
@voidus 
Please review.

**Comment by voidus** [comment]
As far as I can see, this looks good. =)

(I just want to mention again that I couldn't get the tests to work on my machine, so don't take my word for it)

**Comment by blueyed** [comment]
Thanks.

Not even with tox?
E.g. `tox -e py36-dj20-postgres` should work.
OTOH I've just noticed that the envlist was broken, maybe that caused trouble? (ab5821b)

**Comment by voidus** [comment]
Ah, I probably need a database server running, right? I didn't really have the time to look into it, and I don't really want to run a shared postgres server on my workstation.

**Comment by blueyed** [comment]
`tox -e py36-dj20-sqlite` should also work.. :)

**Pull request merged**